src/common/utility/configfile.cpp

```diff
@@ -601,10 +604,13 @@ void FConfigFile::LoadConfigFile ()
 	FileReader file;
 	bool succ;
 
-	FileExisted = false;
+	if (!FileExists(PathName))
+		FileExisted = false;
+
 	if (!file.OpenFile (PathName))
 	{
-		return;
+		if (!FileExisted) return;
+		else I_Error ("Could not Open Config file.\n");
```

Comment:
Could you please put `if` condition and statement on different lines? The same for `else` keyword and its statement.
Character case of the first letters in the error message is a bit random as well.